meta-luneos/recipes-core/base-passwd/base-passwd_%.bbappend

```diff
@@ -19,6 +19,9 @@ do_configure:prepend() {
 #Let's add the system and media users and groups early on as well as the netdev group, since these are needed by multiple recipes.
 
 WEBOS_EXTRA_USERS ?= "system:x:1000:1000::/var:/usr/sbin/nologin \
+                      input:x:1004:1004::/var:/usr/sbin/nologin \
```

Comment:
We should have these already? The patch in this repo fixes input and audio already?

Reply:
Well I have added these because I didn't see them in /etc/passwd ; and (at least on qemux86) the "id" tool doesn't know about the audio user

Reply:
Note that the patch fixes the **groups** IDs, but doesn't handle anything on the user side of things